src/Language/Haskell/TH/Lens.hs

```diff
@@ -2057,15 +2114,32 @@ _LamCaseE
       remitter _ = Nothing
 #endif
 
+-- |
+-- @
+-- _TupE :: 'Prism'' 'Exp' ['Maybe' 'Exp'] -- template-haskell-2.16+
+-- _TupE :: 'Prism'' 'Exp' ['Exp']       -- Earlier versions
+-- @
+#if MIN_VERSION_template_haskell(2,16,0)
+_TupE :: Prism' Exp [Maybe Exp]
+#else
 _TupE :: Prism' Exp [Exp]
+#endif
 _TupE
   = prism' reviewer remitter
   where
       reviewer = TupE
-      remitter (TupE x) = Just x
```

Comment:
This line shouldn't have been removed, right?

Reply:
Yikes, that's definitely a mistake! Can you open a PR with a fix? If not, I can do so later at some point.

Reply:
Resolved in #913, which is present on Hackage as [`lens-4.19.1`](http://hackage.haskell.org/package/lens-4.19.1).